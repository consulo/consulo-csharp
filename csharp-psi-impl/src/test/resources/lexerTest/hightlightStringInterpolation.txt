$"some variable {"dasdas"} asdasdasdas"

$"some variable {'char'} asdasdasdas"

$"some variable {if} asdasdasdas"

$"some variable {$"someValue"} asdasdasdas"

$"some variable {@"someValue"} asdasdasdas"

$"some variable {someValue} asdasdasdas"





