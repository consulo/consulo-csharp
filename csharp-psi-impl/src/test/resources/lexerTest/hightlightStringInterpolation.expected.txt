STRING_INTERPOLATION ('$"some variable {')
STRING_LITERAL ('"dasdas"')
STRING_INTERPOLATION ('} asdasdasdas"')
WHITE_SPACE ('\n\n')
STRING_INTERPOLATION ('$"some variable {'char'} asdasdasdas"')
WHITE_SPACE ('\n\n')
STRING_INTERPOLATION ('$"some variable {if} asdasdasdas"')
WHITE_SPACE ('\n\n')
STRING_INTERPOLATION ('$"some variable {$')
STRING_LITERAL ('"someValue"')
STRING_INTERPOLATION ('} asdasdasdas"')
WHITE_SPACE ('\n\n')
STRING_INTERPOLATION ('$"some variable {@')
STRING_LITERAL ('"someValue"')
STRING_INTERPOLATION ('} asdasdasdas"')
WHITE_SPACE ('\n\n')
STRING_INTERPOLATION ('$"some variable {someValue} asdasdasdas"')